### PR TITLE
LPS-195447 Remove image added by ckeditor

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPS-195447

When pasting Ctrl - V in blogs, the image is added twice. 
This is due to CKEditor adding the second image and so I removed the second image as it is not needed.

Please let me know if there are any questions or comments about this.
Thank you.

### CODE REVIEW

**Review by markocikos — COMMENTED** [review]
@fortunatomaldonado see question inline

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js [hunk 1]
```diff
@@ -417,6 +417,14 @@
...
 							});
 
+							const ckeditorImage = document.querySelector(
+								'[data-cke-saved-src]'
+							);
+
+							if (ckeditorImage) {
+								ckeditorImage.remove();
+							}
+
```
[the remark this answers is not in the document]
Comment (fortunatomaldonado) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js:427]: @markocikos, from my understanding, that image variable is referencing a different image that we want to keep displayed.

**Comment by thektan** [comment]
To prevent a flash of the second image showing before removing, could you try adding CSS like

```css
img[data-cke-saved-src] {
  visibility: hidden;
}
```

At least that's assuming the img with `data-cke-saved-src` is never needed. 🤔

**Comment by fortunatomaldonado** [comment]
Hello @thektan,
I added the CSS and pushed it.
Let me know if there is anything else needed.

Thank you!

**Comment by thektan** [comment]
@fortunatomaldonado Looks good! 👍 

Just curious if there was a reason you only added this for `blogs-web` instead of inside of the `frontend-editor-ckeditor-web` module? https://github.com/liferay-frontend/liferay-portal/blob/baa436a8047384c79ff7ce36aed4bf9dc34c662b/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

**Comment by fortunatomaldonado** [comment]
@thektan, from what I've seen, this is only reproducible in blogs.
If the same issue if found elsewhere, I could go ahead and change it to frontend-editor-ckeditor-web.

**Comment by thektan** [comment]
@fortunatomaldonado Got it, I'm good with this moving forward as is.

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (markocikos) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js:427]: Got it :+1:

**Review by markocikos — APPROVED** [review]
@fortunatomaldonado LGTM! :sparkles:

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
ci:forward

**Comment by JerryAdrianoo** [comment]
Hey, @fortunatomaldonado your changes are probably not related to these issues, but, to be more sure about this, I'll run ci:relevant again and see if it is a fleaky or related.

```
    [junit] Portal log assert failure, see above log for more information: 
    [junit] HHH000315: Exception executing batch [java.sql.BatchUpdateException: Duplicate entry 'uncategorized-20095' for key 'IX_4F90333E'], SQL: insert into ObjectFolder (mvccVersion, uuid_, externalReferenceCode, companyId, userId, userName, createDate, modifiedDate, label, name, objectFolderId) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
    [junit] junit.framework.AssertionFailedError: 
    [junit] HHH000315: Exception executing batch [java.sql.BatchUpdateException: Duplicate entry 'uncategorized-20095' for key 'IX_4F90333E'], SQL: insert into ObjectFolder (mvccVersion, uuid_, externalReferenceCode, companyId, userId, userName, createDate, modifiedDate, label, name, objectFolderId) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
```

**Comment by JerryAdrianoo** [comment]
ci:test:relevant

**Comment by mateusralv** [comment]
Thanks @JerryAdrianoo, I'm on it.

**Comment by mateusralv** [comment]
hello @fortunatomaldonado , I saw that your PR removes the second image of CKeditor, but some tests that add image in the content of blogs are failing in the CI because the image is present but not displayed.

I analyzed all 3 last rounds of Relevant and two tests failed in all and are passing the master.

Tests affected:
LocalFile.BlogsImage#CanViewInlineImageWithPublication
LocalFile.BlogsImage#CanViewInlineImageAfterMovingToFolder

Evidence: 
image

In the content there is an image that is present (I can find it in html) but is not visible. 

[Testray Comparison ](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=3&testrayRunIdB=2430570073&testrayRunIdA=2431940978&statusB=3)

**Comment by fortunatomaldonado** [comment]
@mateusralv, thank you for the investigation! I see what the issue is.

@thektan @markocikos, I updated the CSS since going back to the blog after a save, the image contains `data-cke-saved-src` hiding the image.
Changing it to `img[data-cke-saved-src]:not([data-fileentryid]) {` will only hide the image added by CKEditor.

Let me know if there are any questions or comments about this.
Thank you!

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by mateusralv** [comment]
Hi @fortunatomaldonado Thanks for the explanation I took a look again and the failures now are not related.

[Testray Comparison](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=2432739693&testrayRunId=2432740334)

**Comment by fortunatomaldonado** [comment]
FWD: https://github.com/brianchandotcom/liferay-portal/pull/141520

**Pull request closed without merging**